addons/bus/static/tests/helpers/websocket_event_deferred.js

```diff
@@ -0,0 +1,59 @@
+/* @odoo-module */
+
+import { patchWebsocketWorkerWithCleanup } from "@bus/../tests/helpers/mock_websocket";
+
+import { makeDeferred } from "@web/../tests/helpers/utils";
+import { registerCleanup } from "@web/../tests/helpers/cleanup";
+import { patch, unpatch } from "@web/core/utils/patch";
+
+// Should be enough to decide whether or not notifications/channel
+// subscriptions... are received.
+const TIMEOUT = 500;
+
+/**
+ * Returns a deferred that resolves when the given channel(s) addition/deletion
+ * is notified to the websocket worker.
+ *
+ * @param {string[]} channels
+ * @param {object} [options={}]
+ * @param {"add"|"delete"} [options.operation="add"]
+ *
+ * @returns {import("@web/core/utils/concurrency").Deferred} */
+export function waitForChannels(channels, { operation = "add" } = {}) {
+    const uuid = String(Date.now() + Math.random());
```

Comment:
If the goal is to have it unique, can't we just have a local variable that we increment?

Reply:
Ok for the local variable, it will disappear with the refactoring of the `patch` method anyway!

Reply:
Actually, why not keep it this way? I think the variable approach is more cumbersome, we need to:
- maintain a global variable out of the function
- keep the variable inside the function since we need to known the state of the outer var when the patch was applied.
- we need to add a `QUnit.testStart/Done` to reset it.
All in all this is more than this one liner + it is not destined to be kept.